src/store/modules/auth.store.js

```diff
@@ -50,8 +50,52 @@ export const authStore = {
                         dispatch('alertStore/error', error, { root: true });
                     }
                 );
-        }     
+        },
+
+        checkUser( {dispatch, commit}, userId) {
+            authService.checkUser(userId)
+            .then(
+                () => {
+                    
+                },
+                error => {
+                    commit('loginFailure');
+                    dispatch('alertStore/error', error, { root: true });
+                }
+            ); 
+        },
+            
+        OTP() {
+            authService.OTP()
+        },
+
+        verifyOtp({dispatch}, otp) {
```

Comment:
Please take a look at line number 84 of the same file. See how you have declared `dispatch` there. Why is there such irregularities ?